subjects/blockchain/non-fungible-cats/audit/README.md

```diff
@@ -48,7 +48,7 @@
 
 ##### Buy cat 0 using the buy button
 
-###### Does the buy function work?
+###### Does the buy function works?
```

Comment:
In line 59 there is a missing question mark

Reply:
done